### PR TITLE
manifests: add libsss_sudo package

### PR DESCRIPTION
**Pull request by sohankunkerkar**
Fixes #[445](https://github.com/coreos/fedora-coreos-tracker/issues/445)

### CODE REVIEW

File: manifests/fedora-coreos-base.yaml [hunk 1]
```diff
@@ -129,6 +129,9 @@ packages:
   - cifs-utils
   # Time sync
   - chrony
+  # Allow communication between sudo and SSSD 
+  # for caching sudo rules by SSSD.
```
Comment (dustymabe) [inline manifests/fedora-coreos-base.yaml:133]: I'd support even adding a link to https://github.com/coreos/fedora-coreos-tracker/issues/445 here

**Review by dustymabe — APPROVED** [review]
one comment - otherwise LGTM

File: manifests/fedora-coreos-base.yaml (hunk 1 above)
Comment (sohankunkerkar) [reply manifests/fedora-coreos-base.yaml:133]: done!

**Review by jlebon — APPROVED** [review]

**Comment by jlebon** [comment]
RPM diff sanity check:

```
+ jq .pkgdiff /srv/fcos/builds/latest/x86_64/meta.json
[
  [
    "libsss_sudo",
    0,
    {
      "NewPackage": [
        "libsss_sudo",
        "2.2.3-13.fc31",
        "x86_64"
      ]
    }
  ]
]
```

**Pull request merged**